docs/content/doc/developers/hacking-on-gitea.en-us.md

```diff
@@ -309,6 +309,19 @@ Visual Studio Code. Look at
 [`contrib/ide/README.md`](https://github.com/go-gitea/gitea/blob/main/contrib/ide/README.md)
 for more information.
 
+## GoLand
+
+Clicking the `Run Application` arrow on the function `func main()` in `/main.go` 
+can quickly start a debuggable gitea instance.
+
+The `Output Directory` in `Run/Debug Configuration` MUST be set to the 
+gitea project directory (which contains `main.go` and `go.mod`), 
+otherwise, the started instance's working directory is a GoLand's temporary directory 
+and prevents gitea from loading dynamic resources (eg: templates) in a development environment.  
+
+To run unit tests with SQLite in GoLand, set `-tags sqlite,sqlite_unlock_notify`
+in `Go tool arguments` of `Run/Debug Configuration`.
```

Comment:
Could GoLand be made to just call `make test-backend`? I don't like having these tags in documentation as sooner or later, these tags may change and then this document will be outdated.

Reply:
In vscode I also have to add these tags.

The main benefit is that you can just get editor to run the test individually without having to run through the entire source code tree looking for the matching test name.

It's much much faster

Reply:
I would add that if you're debugging or otherwise running code you will need to set the appropriate build tags anyway. For example if you're debugging gogit.

Reply:
Hmm yeah I guess i agree. Maybe we should at least reference that these default tags come from the Makefile.

Reply:
@silverwind Yes, GoLand can call `make test-backend`, just add an entry in `Run/Debug Configuration`. This guide can run a single unit test, which is very handy and efficient.